internal/infrastructure/host/proxy_infra.go

```diff
@@ -92,7 +93,7 @@ func (i *Infra) runEnvoy(ctx context.Context, out io.Writer, name string, args [
 		defer func() {
 			exit <- struct{}{}
 		}()
-		err := funcE.Run(pCtx, args, funcE.HomeDir(i.HomeDir), funcE.Out(out))
+		err := func_e.Run(pCtx, args, api.HomeDir(i.HomeDir), api.Out(out))
```

Comment:
ps some things I noticed we might want to improve. there is api.EnvoyOut/EnvoyErr which not setting these means they dump to the normal os.Stdout/Stderr which not sure if that's what we want here or not.